src/Views/OperatingSystemView.vala

```diff
@@ -368,10 +380,13 @@ public class About.OperatingSystemView : Gtk.Box {
         try {
             current_state = yield update_proxy.get_current_state ();
         } catch (Error e) {
+            update_stack.visible_child_name = "placeholder";
```

Comment:
Hm any reason not to use what's already there like:
```suggestion
                updates_image.icon_name = "dialog-error";
                updates_title.label = _("System updates not available");
                updates_description.label = _("Couldn't connect to the backend. Try logging out to resolve the issue.");
                button_stack.visible_child_name = "blank";
```

Reply:
Tbh I didn't even think of that :joy: